api/src/test/java/org/pmiops/workbench/utils/mappers/WorkspaceMapperTest.java

```diff
@@ -221,6 +225,86 @@ public void testConvertsFirecloudResponseToApiResponse() {
     assertThat(resp.getWorkspace().getGoogleBucketName()).isEqualTo(FIRECLOUD_BUCKET_NAME);
   }
 
+  @Test
+  public void testConvertsFirecloudResponseToApiResponse_3_param_version() {
```

Comment:
same as the previous test, but for my new 3-param version of toApiWorkspaceResponse()